cypress/integration/fetch_items.js

```diff
@@ -15,6 +17,7 @@ describe("Show list of items", function() {
   describe('List with Items', function() {
     beforeEach(function() {
       window.localStorage.setItem('uniqueToken', 'token1234');
+      cy.addItem('token1234', 'banana', testDate, 2);
```

Comment:
ha, i was just about to make a note on slack about this test as i was worried i wasn't understanding why banana wasn't in there and therefore not understanding this test in our branch. HOWEVER, y'all were already on top of it. Great fix.